drivers/node/ssh/ssh.go

```diff
@@ -106,7 +130,14 @@ func (s *SSH) IsNodeRebootedInGivenTimeRange(n node.Node, timerange time.Duratio
 			Cause: fmt.Sprintf("Failed to run uptime command in node %v", n),
 		}
 	}
-
+	if len(strings.TrimSpace(out.(string))) == 0 {
```

Comment:
This method needs to be executed after node is up so why needs to be wait inside it and not before invoking this method ?